settings/Search.setting.php

```diff
@@ -220,6 +220,19 @@
     'help_text' => NULL,
     'settings_pages' => ['search' => ['weight' => 90]],
   ],
+  'autocomplete_displays' => [
+    'group_name' => 'Search Preferences',
+    'group' => 'Search Preferences',
+    'name' => 'autocomplete_displays',
+    'type' => 'Array',
```

Comment:
Note if switching to a flat list then type is String & options look like  ` 'pseudoconstant' => ['callback' => 'CRM_Core_SelectValues::addressProvider'],`